test_runner/fixtures/zenith_fixtures.py

```diff
@@ -231,10 +409,87 @@ def run(self, arguments: List[str]) -> 'subprocess.CompletedProcess[str]':
 
         return res
 
+    @cached_property
+    def auth_keys(self) -> AuthKeys:
+        pub = (Path(self.repo_dir) / 'auth_public_key.pem').read_bytes()
+        priv = (Path(self.repo_dir) / 'auth_private_key.pem').read_bytes()
+        return AuthKeys(pub=pub, priv=priv)
+
 
 @zenfixture
-def zenith_cli(zenith_binpath: str, repo_dir: str, pg_distrib_dir: str) -> ZenithCli:
-    return ZenithCli(zenith_binpath, repo_dir, pg_distrib_dir)
+def shared_simple_env(request: Any, port_distributor) -> Iterator[ZenithEnv]:
```

Comment:
`zenith_shared_simple_env` for consistency? Or even `_shared_simple_env` to highlight it's an internal-only fixture.